src/Compilers/CSharp/Test/Symbol/Symbols/Source/NamedTypeTests.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class NamedTypeTests : CSharpTestBase
```

Comment:
The name of the test class feels oddly general and to me it doesn't evoke what these tests are about. Consider a name like `ExplicitNoPiaLocalTypeTests`, or something of that nature.

Reply:
I could.  This felt like  areasonable place for any NamedTypeTests (which also matches the names of the siblings) where the nopia tests were just a subset.  if you prefer a dedicated test type jsut for these, that's fine with me.

Reply:
If that's the ultimate intention for the class that seems fine. I suspect that in practice people looking for the relevant tests might just find-all-refs on the relevant symbol APIs anyway.